src/openrct2/peep/Guest.cpp

```diff
@@ -1546,17 +1546,19 @@ bool Guest::DecideAndBuyItem(Ride* ride, int32_t shopItem, money32 price)
     }
 
     if (shopItem == SHOP_ITEM_UMBRELLA && climate_is_raining())
-        goto loc_69B119;
-
-    if ((shopItem != SHOP_ITEM_MAP) && ShopItems[shopItem].IsSouvenir() && !hasVoucher)
+    {
+        //goto loc_69B119;  replaced by "else if" below
+        std::cout << "It's raining and I'm considering an umbrella\n";
```

Comment:
Uhm, is this leftover test code?

Also, you can put the two if togethers:
```
auto consideringUmbrella = shopItem == SHOP_ITEM_UMBRELLA && climate_is_raining();
if (!consideringUmbrella && (shopItem != SHOP_ITEM_MAP) && ShopItems[shopItem].IsSouvenir() && !hasVoucher)
```

Also remove the `//loc_69B119: replaced by "else" above` comment.

Reply:
OK, I will clean it up.